README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
All tge configuration mumbo-jumbo: the servicemanager just accepts ctor config now